expr.go

```diff
@@ -84,6 +94,55 @@ func CompileType(input string, ops ...OptionFn) (*vm.Program, reflect.Type, erro
 	return program, t, nil
 }
 
+type operatorPatcher struct {
```

Comment:
Let's move it to another package, this top level package is for reexporting only. Maybe move it to `checker` or to new package (like `postprocessing`)